src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,59 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+interface Tab {
+  id: string;
+  title: string;
+  content: string;
+}
+
+interface Props {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+}
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const findselectedTabId = tabs
+    .find(tab => tab.id === selectedTabId) || tabs[0];
```

Comment:
It's a whole tab, not just id
```suggestion
  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```